x-pack/metricbeat/module/cockroachdb/status/manifest.yml

```diff
@@ -4,3 +4,16 @@ input:
   metricset: collector
   defaults:
     metrics_path: /_status/vars
+    use_types: true
+processors:
+  - drop_fields:
```

Comment:
Wondering if this could make use of [`metrics_filters`](https://www.elastic.co/guide/en/beats/metricbeat/7.7/metricbeat-metricset-prometheus-collector.html#_filtering_metrics_2) instead?

Reply:
We could, as all `sql_mem_sql...` metrics are duplicated in cockroachdb. But only histograms are problematic, the rest are gauges, and in principle Metricbeat doesn't have any problem with duplicated gauges.
Also, removing these metrics at this point ensures that they are never collected, even if the user sets its own `metrics_filters`, or adds its own processors.